sqlglot/dialects/exasol.py

```diff
@@ -173,6 +174,8 @@ def datatype_sql(self, expression: exp.DataType) -> str:
                     self, e, func_name="INSTR", supports_position=True, supports_occurrence=True
                 )
             ),
+            # https://docs.exasol.com/db/latest/sql_references/functions/alphabeticallistfunctions/hash_sha%5B1%5D.htm#HASH_SHA%5B1%5D
+            exp.SHA: rename_func("HASH_SHA1"),
```

Comment:
Should this map to `HASH_SHA` instead of `HASH_SHA1`? From Exasol's docs:

> `HASH_SHA()` is an alias for `HASH_SHA1()`.

Reply:
Thanks for pointing I just checked it again either of them is correct. HASH_SHA is better